DB/src/main/java/io/deephaven/db/v2/utils/UpdatePerformanceTracker.java

```diff
@@ -178,6 +178,8 @@ private void finishInterval(final long intervalStartTimeMillis, final long inter
 
         boolean encounteredErrorLoggingToMemory = false;
 
+        long maxStartTimeNanos = 0;
```

Comment:
These are never updated. It suggests to me that some part of this change is incomplete.

Reply:
You are correct, see next answer.